reason-graphql/src/language/Graphql_Language_Parser.re

```
@@ -21,6 +21,12 @@ let expect = (lexer: Lexer.t, token: Lexer.token) =>
   | currToken when currToken == token => Lexer.advance(lexer)
   | _ => expectedError(lexer, token)
   };
+  
+let expectOptional = (lexer: Lexer.t, token: Lexer.token) =>
```

Thinking about this now...may cause issues since it's essentially hiding `Error`'s right?

---

Do you want to swallow the error here? It seem like you want to advance the token

---

Is this the same as skip (couple lines below)?